release-notes/1.19.2.md

```diff
@@ -0,0 +1,26 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* When the Anbox Cloud Appliance upgrade fails, you can force run the upgrade again by using the `--force` option when running `anbox-cloud-appliance upgrade`.
+* Android security updates for October 2023 (see [Android Security Bulletin - October 2023](tbd) for more information).
+* The Android WebView has been updated to [tbd](tbd).
```

Comment:
I don't think we update the WebView version for .2 releases, but I may be wrong.

Reply:
We do if there is an update. 1.19 was updated to `117.0.5938.61` already.

Reply:
@morphis Wouldn't we pick 117.0.5938.153 as per https://chromereleases.googleblog.com/2023/10/chrome-for-android-update.html?